go/cmd/vtctldclient/command/serving_graph.go

```diff
@@ -140,6 +148,35 @@ func commandGetSrvKeyspaces(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+var updateThrottlerConfigOptions = struct {
+	Enable            bool
+	Disable           bool
+	Threshold         float64
+	CustomQuery       string
+	CheckAsCheckSelf  bool
+	CheckAsCheckShard bool
+}{}
+
+func commandUpdateThrottlerConfig(cmd *cobra.Command, args []string) error {
+	cli.FinishedParsing(cmd)
+
+	_, err := client.UpdateThrottlerConfig(commandCtx, &vtctldatapb.UpdateThrottlerConfigRequest{
+		Enable:            updateThrottlerConfigOptions.Enable,
+		Disable:           updateThrottlerConfigOptions.Disable,
+		Threshold:         updateThrottlerConfigOptions.Threshold,
+		CustomQuery:       updateThrottlerConfigOptions.CustomQuery,
+		CheckAsCheckSelf:  updateThrottlerConfigOptions.CheckAsCheckSelf,
+		CheckAsCheckShard: updateThrottlerConfigOptions.CheckAsCheckShard,
+	})
+	if err != nil {
+		return err
+	}
+
+	fmt.Println("UpdateThrottlerConfig: ok")
```

Comment:
do we need this?

Reply:
This seems consistent with other functions. I only added it because other functions do this.

Reply:
removed